prow/pod-utils/decorate/podspec.go

```diff
@@ -243,6 +244,11 @@ func ProwJobToPodLocal(pj prowapi.ProwJob, outputDir string) (*coreapi.Pod, erro
 		if len(container.TerminationMessagePolicy) == 0 {
 			spec.Containers[i].TerminationMessagePolicy = coreapi.TerminationMessageFallbackToLogsOnError
 		}
+		if pj.Spec.DecorationConfig != nil && pj.Spec.DecorationConfig.SetLimitEqualsMemoryRequest != nil && *pj.Spec.DecorationConfig.SetLimitEqualsMemoryRequest && container.Resources.Requests != nil {
+			if val, ok := container.Resources.Requests[v1.ResourceMemory]; ok {
+				spec.Containers[i].Resources.Limits[v1.ResourceMemory] = val
+			}
+		}
```

Comment:
I think we should do this with the rest of the decoration in the `decorate()` function.

Reply:
interesting enough that I thought I did it on `decorate()` method, fixed now